llvm/test/Transforms/InstCombine/pr96012.ll

```diff
@@ -0,0 +1,24 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S -passes=instcombine < %s | FileCheck %s
+
+target datalayout = "e-m:e-p:64:64-i64:64-i128:128-n32:64-S128"
```

Comment:
Needed?

Reply:
Yes. `datalayout` is needed to reproduce this issue: https://godbolt.org/z/7hxP6dx1j